standard-interfaces/cloud.astarte.genericsensors.SamplingRate.json

```diff
@@ -0,0 +1,25 @@
+{
+    "interface_name": "cloud.astarte.genericsensors.SamplingRate",
+    "version_major": 0,
+    "version_minor": 1,
+    "type": "properties",
+    "ownership": "server",
+    "description": "Configure sensors sampling rate and enable/disable.",
+    "doc": "This interface allows to set generic sensors sampling rate and enable/disable policies for each sensor. Sensors are identified by their sensor_id. See also cloud.astarte.genericsensors.Values.",
+    "mappings": [
+        {
+            "endpoint": "/%{sensor_id}/enable",
+            "type": "boolean",
+            "allow_unset": true,
+            "description": "Enable/disable sensor data transmission.",
+            "doc": "When true sampled data transmission is always on, otherwise when false is off. When unset data transmission policy is up to the sensor."
+        },
+        {
+            "endpoint": "/%{sensor_id}/samplingPeriod",
+            "type": "integer",
+            "allow_unset": true,
+            "description": "Sensor sample transmission period.",
+            "doc": "Send a sampled value every samplingPeriod seconds. When unset sampling period is up to the sensor."
```

Comment:
probably millisecond and longinteger?

Reply:
We are trying to encourage people to use sane sampling rates, such as 5 seconds, instead of 5 milliseconds. If this doesn't fit some user requirements the other 2 interfaces are still usable and this interface can be replaced with a custom one.